app/portainer/views/endpoints/edit/endpoint.html

```diff
@@ -124,6 +124,14 @@
           </div>
 
           <!-- !endpoint-public-url-input -->
+
+          <tls-fieldset
+            ng-if="!state.edgeEndpoint && endpoint.Status !== 4 && state.showTLSConfig"
+            values="formValues.tlsConfig"
+            on-change="(onChangeTLSConfigFormValues)"
+            validation-data="{optionalCertValidation: state.allowOptionalCertValidation}"
```

Comment:
```suggestion
            validation-data="{optionalCertValidation: true}"
```

why do we need an extra variable for this?